packages/mobile/src/common/constants/constants.ts

```diff
@@ -1,6 +1,25 @@
 import { Platform } from 'react-native';
+import { CameraOptions, ImageLibraryOptions } from 'react-native-image-picker';
 
 const IS_ANDROID = Platform.OS === 'android';
 const IS_IOS = Platform.OS === 'ios';
 
 export { IS_ANDROID, IS_IOS };
+
+export const IMAGE_OPTIONS: ImageLibraryOptions = {
```

Comment:
Let's extract these two config constants to a separate file within constants folder

Reply:
updated